static/locales/en/translation.json

```diff
@@ -0,0 +1,70 @@
+{
+  "css": {
+    "missing-curly": "You have a starting { but no ending } to go with it.",
+    "property-missing-colon": "Put a colon (:) between the property and the value.\nTry: color: red",
+    "selector-missing": "Start every block of CSS with a selector, such as an element name or class name.\nTry: p {\n  color: red;\n}"
+  },
+  "prettycss": {
+    "block-expected": "Start a block using { after your selector.\nTry: __error__ {",
+    "extra-tokens-after-value": "Looks like you're missing a semicolon on the line before this one.",
+    "illegal-token-after-combinator": "After a + or > in a selector, you need to specify the name of another element, class, or ID",
+    "invalid-token": "This line doesn't look like valid CSS.",
+    "invalid-value": "__error__ isn't a meaningful value for this property. Double-check what values you can use here.",
+    "require-value": "Put a value for __error__ after the colon.",
+    "selector-expected": "Use a comma to separate multiple tag names, classes, or IDs.",
+    "unknown-property": "__error__ isn't a property that CSS understands. Double-check the name of the property that you want to use."
+  },
+  "html": {
+    "align": "Don't use the \"align\" attribute. Instead, use the CSS text-align property",
+    "background": "Don't use the \"background\" attribute. Instead, use the CSS background property",
+    "bgcolor": "Don't use the \"bgcolor\" attribute. Instead, use the CSS background-color property",
+    "border": "",
+    "frameborder": "Don't use the \"__attribute__\" attribute. Instead, use the CSS border property",
+    "marginwidth": "Don't use the \"marginwidth\" attribute. Instead, use the CSS margin-left and margin-right properties",
+    "marginheight": "Don't use the \"marginheight\" attribute. Instead, use the CSS margin-top and margin-bottom properties",
+    "scrolling": "Don't use the \"scrolling\" attribute. Instead, use the CSS overflow property",
+    "width": "Don't use the \"width\" attribute. Instead, use the CSS width property",
+    "E002": "Use lower case for attribute names",
```

Comment:
I think we should keep the knowledge of error codes in the `validation` module itself—so the keys here would be semantic ones, like `lower-case-attribute` in this case.